configs/mini_pupper_config/package.xml

```diff
@@ -11,10 +11,9 @@
 
   <buildtool_depend>ament_cmake</buildtool_depend>
   <build_depend>launch_ros</build_depend>
-  <!-- <exec_depend>roslaunch</exec_depend>
-  <exec_depend>rviz</exec_depend>
   <exec_depend>champ_base</exec_depend>
-   -->
+  <exec_depend>launch_ros</exec_depend>
```

Comment:
do we need this? 
i didnt see chvmp projects put it like this

Reply:
Yes I think we need this because `bringup.launch.py` imports `launch` and `launch_ros`. Maybe we should send a PR to add `launch_ros` in chvmp projects.
Also added `launch` package in https://github.com/CullenSUN/chvmp_robots/pull/1/commits/8545c796cbfff491358f99ab2259a838b644fa65

ref: https://github.com/ros2/demos/blob/galactic/dummy_robot/dummy_robot_bringup/package.xml